app/uk/gov/hmrc/individualsemploymentsapi/service/v2/ScopesService.scala

```diff
@@ -35,13 +35,13 @@ class ScopesService @Inject()(configuration: Configuration) {
     getScopeItemsKeys(scope)
       .flatMap(fieldId => apiConfig.endpoints.flatMap(e => e.fields.get(fieldId)))
 
-  private[service] def getEndpointFieldKeys(endpointKey: String): Iterable[String] =
```

Comment:
I'm not keen on making these public, but I had a quick play with it myself and couldn't fix the compiler issues in the tests...